contrib/atuin/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "atuin"
+pkgver = "18.1.0"
+pkgrel = 0
+build_wrksrc = "atuin"
+build_style = "cargo"
+hostmakedepends = ["cargo"]
+makedepends = ["sqlite-devel"]
```

Comment:
should also add a patch like

```
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,12 +33,12 @@
 
 [workspace.dependencies.reqwest]
 version = "0.11"
-features = ["json", "rustls-tls-native-roots"]
+features = ["json", "default-tls"]
 default-features = false
 
 [workspace.dependencies.sqlx]
 version = "0.6"
-features = ["runtime-tokio-rustls", "chrono", "postgres"]
+features = ["runtime-tokio-native-tls", "chrono", "postgres"]
 
 [patch.crates-io]
 sqlx = { path = "./vendor/sqlx" }
```

to use openssl instead

Reply:
I do understand patching tools to use system libraries instead of bundled versions of that same library, but why replace rustls with openssl?